plugin/pxr/maya/lib/usdMaya/testenv/testUsdMayaAdaptorUndoRedo.py

```diff
@@ -0,0 +1,80 @@
+#!/pxrpythonsubst
+#
+# Copyright 2020 Pixar
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import mayaUsd.lib as mayaUsdLib
+
+from pxr import UsdGeom
+
+from maya import cmds
+from maya import standalone
+
+import unittest
+
+
+class testUsdMayaAdaptorUndoRedo(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        standalone.initialize('usd')
+
+        # We load the pxrUsd plugin here to ensure that the usdUndoHelperCmd
```

Comment:
We would like to move / add tests which can work with either plugin outside of `plugins/...` and into `test/...`. I believe both testUsdMayaAdaptor and testUsdMayaAdaptorUndoRedo fall into the above category?

Reply:
@kxl-adsk: Sounds good. I just rebased the original change as https://github.com/Autodesk/maya-usd/pull/694/commits/a95b06260fd9b9e8376f6ff3f17c5b4cedfde638, and then migrated all of the `UsdMayaAdaptor` tests to `test/lib/usd/translators`. I moved them all since it seems like they should all stay together, and the ones that need plugins loaded will work with either `pxrUsd` or `mayaUsdPlugin`.

I had to fix one thing in `fixturesUtils.py` which I mentioned in https://github.com/Autodesk/maya-usd/pull/613#issue-442143534, which is to only create aliases for the export and import commands if we're actually loading the plugin. Otherwise the tests where we're specifically trying *not* to load plugins would fail there. That fix is in https://github.com/Autodesk/maya-usd/pull/694/commits/c56253f496488e3c0ce5647646e72ebc21460b74.